### DISCUSSION TITLE
Import Existing Mesh

### DISCUSSION THREAD

**Discussion by embraceentropy** [jneilliii/OctoPrint-BedLevelVisualizer#423] [Q&A] [answered]
Just installed this plugin. I already have a manual mesh in EEPROM. Is there a way to retrieve this data for visualization or do I have to perform a fresh manual mesh?

**Comment by jneilliii** [comment, marked as answer]
You can typically use `M420 V` to retrieve EEPROM mesh, that's what I use with UBL.

**Reply by embraceentropy** [reply]
Thank you!!
